.github/workflows/follow-merge-upstream-repo-sync.yml

```diff
@@ -161,11 +161,12 @@ jobs:
 
       - name: "LSF Docs: Install NPM deps"
         if: github.event.client_payload.repo_name == 'heartexlabs/label-studio-frontend'
-        working-directory: tmp/scripts
-        run: npm install jsdoc-to-markdown node-fetch
+        continue-on-error: true
+        run: npm install -g jsdoc-to-markdown node-fetch
```

Comment:
These modules are installed for main LSF code, so they should be accessible already. Try to remove this step completely and check Docs Build